core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -738,6 +738,90 @@ public MetricKey build() {
           .setMetricType(MetricType.COUNTER)
           .build();
 
+  // Distributed command related metrics
```

Comment:
one high level comment here, it might be easier if we programmatically construct these MetricKey's so that if we decide to add another JobType it will be easier to add?

Reply:
Wrap up in this PR or leave it to be done in a next PR when adding for new JobTypes?

Reply:
sounds good.